flow-typed/npm/remotedev-serialize_vx.x.x.js

```diff
@@ -0,0 +1,50 @@
+// flow-typed signature: bf2231eb3ff430dbd168fd5d3b535a03
+// flow-typed version: <<STUB>>/remotedev-serialize_v0.1.8/flow_v0.92.0
+
+/**
+ * This is an autogenerated libdef stub for:
+ *
+ *   'remotedev-serialize'
+ *
+ * Fill this stub out by replacing all the `any` types.
+ *
+ * Once filled out, we encourage you to share your work with the
+ * community by sending a pull request to:
+ * https://github.com/flowtype/flow-typed
+ */
+
+declare module 'remotedev-serialize' {
+  /**
+   * Flowtype definitions for index
+   * Generated by Flowgen from a Typescript Definition
+   * Flowgen v1.10.0
+   */
```

Comment:
Interesting!

Where did this file come from -- did you have to run Flowgen yourself (and how)? Or did you run something like `yarn update-libdefs` and find that that ran Flowgen automatically?

Reply:
... I'm also kind of puzzled because I don't see any TypeScript in that library!
```
$ find node_modules/remotedev-serialize/
node_modules/remotedev-serialize/
node_modules/remotedev-serialize/README.md
node_modules/remotedev-serialize/helpers
node_modules/remotedev-serialize/helpers/index.js
node_modules/remotedev-serialize/immutable
node_modules/remotedev-serialize/immutable/serialize.js
node_modules/remotedev-serialize/immutable/index.js
node_modules/remotedev-serialize/package.json
node_modules/remotedev-serialize/index.js
node_modules/remotedev-serialize/constants
node_modules/remotedev-serialize/constants/options.js
```

So I wonder where the information is coming from.

Reply:
Ah! It comes from DefinitelyTyped, and I did indeed run Flowgen myself. I'll get this all meticulously documented in the next revision.